themes/bootstrap3/templates/Recommend/LibGuidesAZResults.phtml

```diff
@@ -0,0 +1,19 @@
+<?php
+  $searchObject = $this->recommend->getResults();
+  $results = $searchObject->getResults();
+?>
+<?php if (!empty($results)): ?>
+  <h4><?=$this->transEsc('libguidesaz_recommendations')?></h4>
+  <div class="list-group">
+    <?php foreach ($results as $driver): ?>
+      <div class="list-group-item">
+        <span>
+          <a href="<?=$driver->getUniqueID()?>" class="title">
```

Comment:
Is it legit to go straight to $driver for this, rather than through some view helper method?  I realize it's weird that the unique ID is the URL, but that's what the driver does, inherited from the original LibGuides driver.  It kind of makes sense as there is very little data retrieved from the data source.

Reply:
Done